migration_sql_test.go

```diff
@@ -113,21 +116,54 @@ func TestUseTransactions(t *testing.T) {
 		if err != nil {
 			t.Error(err)
 		}
-		_, useTx := getSQLStatements(f, true)
+		_, useTx, err := getSQLStatements(f, true)
+		if err != nil {
+			t.Error(err)
+		}
 		if useTx != test.useTransactions {
 			t.Errorf("Failed transaction check. got %v, want %v", useTx, test.useTransactions)
 		}
 		f.Close()
 	}
 }
 
+func TestParsingErrors(t *testing.T) {
+	type testData struct {
+		sql           string
+		direction     bool
+		expectedError string
+	}
+	tests := []testData{
+		{
+			sql:           statementBeginNoStatementEnd,
+			direction:     true,
+			expectedError: "parsing migration: saw '-- +goose StatementBegin' with no matching '-- +goose StatementEnd'",
```

Comment:
Let's not test for specific error string. Just check for `nil`.

`error: true`